partner_statement/README.rst

```diff
@@ -0,0 +1,132 @@
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--reporting-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-reporting/tree/12.0/partner_statement
+    :alt: OCA/account-financial-reporting
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-reporting-12-0/account-financial-reporting-12-0-partner_statement
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/91/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of Invoicing to support the printing of customer and vendor statements. 
+There are two types of statements, Activity and Outstanding.  Aging details can be shown in the reports, expressed in aging buckets,
+so the customer or vendor can review how much is open, due or overdue.
+
+The activity statement provides details of all activity on the partner receivables or payables
+between two selected dates. This includes all invoices, refunds and payments.
+Any outstanding balance dated prior to the chosen statement period will appear
+as a forward balance at the top of the statement. The list is displayed in chronological
+order and is split by currencies.
+
+The outstanding statement provides details of all outstanding partner receivables or payables
+up to a particular date. This includes all unpaid invoices, unclaimed refunds and
+outstanding payments. The list is displayed in chronological order and is split by currencies.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+
+Users willing to access to this report should have proper Accounting & Finance rights:
+
+#. Go to *Settings / Users* and edit your user to add the corresponding access rights as follows.
+#. In *Application / Accounting & Finance*, select *Billing* or *Billing Manager*
+
+To configure this module, you need to:
+
+#. Go to *Configuration / Settings*
+#. Under the *Followup Section* of *Accounting* option select either or both of OCA Activity or Outstanding Statement
+#. Once selected, you may set default options for the reports.
+#. Click *Save*
+
+Removing the wizard from menus follows the same process.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Invoicing > Sales > Master Data > Customers or Invoicing > Purchases > Master Data > Vendors and select one or more
```

Comment:
Could you update this line? I think in v12 this is different...

Reply:
Well it is different in Enterprise as well. Just made it generic. You can run this from any list of partners.

Reply:
haha good